develop.sh

```diff
@@ -6,6 +6,6 @@ trap 'kill 0' EXIT
 (cd static-site && npm run develop) &
 
 # Start Express server with proxying of Gatsby assets to localhost:8000
-NODE_ENV="dev" GATSBY_DEV_URL=http://localhost:8000 node server.js --verbose
+NODE_ENV="dev" GATSBY_DEV_URL=http://localhost:8000 ./node_modules/.bin/nodemon --verbose -- server.js --verbose
```

Comment:
`npx` does two main things we might care about before exec-ing:

1. Potentially auto-install, which we don't really want here.
2. Add `./node_modules/.bin` to `PATH`, which we don't need here.

`nodemon` itself also does (2), so the only effective difference is the lack of auto-install.

Reply:
I should also note that other existing places in the codebase invoke programs in `./node_modules/.bin/` directly instead of via `npx`, e.g. our `build.sh`.

Reply:
Noting that (1) can be ignored with `npx --no`, but I'm in favor of what's here for clarity and consistency.